wurst/network/SyncSimple.wurst

```diff
@@ -0,0 +1,170 @@
+package SyncSimple
+
+import MapBounds
+import UnitIndexer
+import ErrorHandling
+import BitSet
+import LinkedListModule
+
+/********************************************************************************
+	SyncSimple library v1.0 by MoriMori.
+
+	Based on TriggerHappy's SyncInteger library.
+
+	Rationale:
+		This library can be used to 'synchronize' players after some async network
+		action has been performed by one or more players.
```

Comment:
This is perhaps a misnomer, but 'syncing' in WC3 is commonly referring to the act not of recovering from desyncs, but rather sending local data from one player to the rest, so that it can be safely used in synchronous code.

One such example is the various natives that do not take a Player parameter, like GetCameraPos. It is asynchronous and would desync if you used it in synchronous code to get the position of some player's camera. However, with this, you can send one player's camera position to another safely without risking a desync.

Another much more useful example is in save/load systems that work with files. Since file data is only available to one player, we need to sync it to others as well.

When I finish my MultifileIO library, it will be much more obvious! I will also provide various test cases soon for Network library soon when I have the time, that can serve as usage examples.

Reply:
It would be great to have some amount of this explanation instead of just "This library can be used to 'synchronize' players after some async network action has been performed" which sounds like "I did some magical desync thing which I can't debug and now I'm using Network to fix my problems".